backend/test/card_game/hand_test.clj

```diff
@@ -36,14 +36,17 @@
 (expect
   #(= (count (:hand %)) 11)
   (-> (new-game)
+      (play-card 0 2 1)
       (play-card 1 1 1)
       :players
       second))
 
 ; Playing a card does not affect other players' hand
```

Comment:
This title comment (Playing a card does not affect other players' hand) can be tested only (as far as I know) by giving the played cards different powers in hand through `(alter-card`.

Reply:
Done. It seems pretty similar to other tests on `effects_test.clj` though.

Reply:
You're right, we'll probably remove this test on the next cleanup phase